### ISSUE TITLE
Old BTW textures and a full 1.16 version of bedrock (as far as i can tell =P)

### ISSUE THREAD

**Issue by pfccalico** [SummerFields/SummerFields#675]
[Summerfield.zip](https://github.com/SummerFields/SummerFields/files/4976103/Summerfield.zip)

[BTW Support.zip](https://github.com/SummerFields/SummerFields/files/4976104/BTW.Support.zip)

Just a couple of zip files, the first one is one i originally downloaded from planetminecraft and updated to 1.16, i wasn't aware you guys were working on it as well so i kinda finished what chengzzi started.

The second zip file is the better than wolves support zip for whatever alternatives you might want or inspiration. also here is the cobble ctm i did many years ago on the classic summerfields cobblestone texture. dunno how that could work now though?

![14](https://user-images.githubusercontent.com/8011723/88462961-44f52c00-cea7-11ea-90f8-ccca1ebffe2d.png)
![15](https://user-images.githubusercontent.com/8011723/88462962-458dc280-cea7-11ea-8f3d-f942ef272082.png)
![12](https://user-images.githubusercontent.com/8011723/88462964-458dc280-cea7-11ea-98a0-b57a0d038fb6.png)
![13](https://user-images.githubusercontent.com/8011723/88462965-458dc280-cea7-11ea-91c6-0979d934613b.png)

**Comment by tobyplowy** [comment]
We are the official team of summerfields. 
Some good things in there. @lipki Opinions?

**Comment by pfccalico** [comment]
i have another zip with a full summerfields resource pack that also has better than wolves textures. It can be found here 

http://www.mediafire.com/file/6shqbh8bbc69st6/SF1.5.1_MOGctm.zip/file

**Comment by pfccalico** [comment]
![plate_1](https://user-images.githubusercontent.com/8011723/88463158-a964bb00-cea8-11ea-906c-59177320a625.png)
![plate_2](https://user-images.githubusercontent.com/8011723/88463159-aa95e800-cea8-11ea-9674-bf5b4934b3e5.png)

**Comment by pfccalico** [comment]
a little similar to the diamond and iron combined but maybe tonally different enough to not be the same? i know you are looking at the netherite as being done? but personally i think it's too colourful for something found in the nether and the style doesn't match the theme of the pack (in my opinion).

**Comment by pfccalico** [comment]
could maybe edge the plates in "gold" to make it more ornate

**Comment by pfccalico** [comment]
infact, thats something ill get on to and put here as a possible alternative or something to be used on a mod support pack?

**Comment by tobyplowy** [comment]
I like that idea. 
We may be able to use it as the netherite texture for armor.

**Comment by lipki** [comment]
Hi pfccalico, glad to see a former contributor here.
And thank you for your files.
An independent update of the pack, it must be a good idea, I can not wait to see that.
I imagine that we will be faced with difficult choices.

We can't manage mods with the current team.
But I do intend to redeliver the Optifine mods one day anyway.

**Comment by lipki** [comment]
( I don't consider the netherite stuff finished. )

**Comment by lipki** [comment]
I like the mystical side of the current netherite stuff.
But the armor is not correct.

I like your armor, more like the original.
But not really original: D

**Comment by pfccalico** [comment]
yeah i know, i was trying to throw stuff together back then to get support for the mod at the time. I'll chuck together a version of the netherite armour that is a little more original than that lol all i did at the time was grey scale and darken the armour for the iron set and close the helm. I will put something here once i have a design

**Comment by pfccalico** [comment]
although i like the copper tones of the current netherite armour, it does remind me a bit of samus though XD

**Comment by lipki** [comment]
If you want to continue to take care of the Bedrock
the repo is here https://github.com/SummerFields/MCPE

**Comment by pfccalico** [comment]
welp

here's my attempt at netherite. What do you think?

![netherite_2](https://user-images.githubusercontent.com/8011723/88465345-3106f580-ceba-11ea-8b63-ecf737319dd4.png)
![netherite_1](https://user-images.githubusercontent.com/8011723/88465346-319f8c00-ceba-11ea-8235-5e7453f9b296.png)

**Comment by pfccalico** [comment]
![image](https://user-images.githubusercontent.com/8011723/88465400-ac68a700-ceba-11ea-8e2a-47f4b00917b5.png)

**Comment by pfccalico** [comment]
![image](https://user-images.githubusercontent.com/8011723/88465418-c99d7580-ceba-11ea-8cb5-b5b9c1b77129.png)

**Comment by pfccalico** [comment]
its just a little something i threw together using elements from all the other armours

sooooooo...... still not original XD

**Comment by lipki** [comment]
No it's fine.
Post here #602 we will think about it.
